terraform/trial_rg/modules/fhir/main.tf

```diff
@@ -27,17 +27,28 @@ resource "azurerm_app_service" "fhir_server" {
 
   site_config {
     linux_fx_version = "DOCKER|${var.fhir_image_name}:${var.fhir_image_tag}"
+    always_on        = true
+  }
+
+  logs {
+    http_logs {
```

Comment:
what about application logs and failed requests? may want these as well

Reply:
agree